src/test/java/com/easyrestaurant/tests/CreateAdministratorProfileTest.java

```diff
@@ -0,0 +1,74 @@
+package com.easyrestaurant.tests;
+
+import java.sql.SQLException;
+
+
+import com.easyrestaurant.core.TestUtils;
+import com.easyrestaurant.utils.CheckRestaurantExist;
+import com.easyrestaurant.utils.LoadProperties;
+import io.qameta.allure.Description;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class CreateAdministratorProfileTest extends TestUtils {
+	
+//	WebDriver driver;
+	static LoadProperties properties = new LoadProperties();
+//	static CheckRestaurantExist restaurant = new CheckRestaurantExist();
+
+	@Parameters({"username", "password"})
```

Comment:
Why do you have @Parameters annotation when you don't use the values?

Reply:
Because I get error with message "missing parameters".

Reply:
Please ignore this parameters. It does not stop you to run the test.